### PR TITLE
Updated documentation for isVisible(). Fixes #86

### LINKED ISSUE #86: Documentation bug -   waitForIsNotVisible(); waitIsVisible();
**Describe the bug**

This description is not sufficient to actually understand

    /**
     * Wait till element is not visible within the viewport (--> What does the boolean parameter do?)
     */
    boolean waitForIsVisible(final boolean complete);

    /**
     * Wait till element is not more visible within the viewport
     */
    boolean waitForIsNotVisible();

**To Reproduce**
Look at 1.0-RC-32 eu/tsystems/mms/tic/testframework/pageobjects/internal/waiters/GuiElementWait.java

**Expected behavior**
Clear documentation on what the parameters do in this context and what the different methods actually do. There is also no documentation here: http://docs.testerra.io/testerra/1.0-RC-23/#_create_a_new_project

**Environment(please complete the following information):**
 - OS: MacOS
 - Browser n/a
 - Version 1.0-RC-32

**Additional context**
I have a problem that the following code does not get me a click in the TAP:

cartLink.asserts().assertIsDisplayed();
cartLink.click();

https://tapbs01.t-systems-mms.eu/repository/download/STOcep_STOcepTesterraConfigChromeNightly/548676:id/test-report/report-ng/index.html#/method/60decb1e7da0effbc6056bfc/details I am trying to find a way to make my test more rubust and thought waitForIsVisible(); could help, but I do not understand how to use it.

### PR DESCRIPTION
**Pull request by mreiche**
# Description

Documentation updates

Fixes #86

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/src/docs/guielement/guielement-checks.adoc [hunk 1]
```diff
@@ -18,15 +18,15 @@ and it's `width` and `height` are both greater than 0.
 element.isDisplayed();
 ----
 
-Checks if the element is displayed and if it's partially or `complete` visible
-in the scroll area of the Viewport.
+Checks if the element is displayed and if it's partially or `fully` visible
```
Comment (martingrossmann) [inline docs/src/docs/guielement/guielement-checks.adoc:21]: I think you can remove the code markers around fully. In the version before it points to the parameter which had the same name.

**Review by martingrossmann — APPROVED** [review]

**Review by mbeuthan — APPROVED** [review]

**Comment by chrberSTO** [comment]
Where can I see, which release this bugfix will be part of?

**Comment by martingrossmann** [comment]
> Where can I see, which release this bugfix will be part of?

Testerra 1.4 was released some days ago. This fix will be part of 1.5, but there is no time plan for it. Of course you can view into the sources. Github supports the visualization of Asciidoc files.

**Comment by mreiche** [comment]
@chrberSTO Please send me login credentials to the SUT and I can proove if the bug is fixed in latest upstream.

**Comment by chrberSTO** [comment]
Hi Mike,

 

If you like, you can instruct me how to get the latest upstream and I could check, if the problem is gone.

Will have to check with company about the credentials.

 

Thanks.

Christian

 

From: Mike Reiche ***@***.***>
Reply to: telekom/testerra ***@***.***>
Date: Tuesday, 6. July 2021 at 09:06
To: telekom/testerra ***@***.***>
Cc: chrberSTO ***@***.***>, Mention ***@***.***>
Subject: Re: [telekom/testerra] Updated documentation for isVisible(). Fixes #86 (#87)

 

@chrberSTO Please send me login credentials to the SUT and I can proove if the bug is fixed in latest upstream.

**Comment by mreiche** [comment]
I give you instructions here: https://github.com/telekom/testerra/issues/86 Lets keep this closed here.

**Pull request merged**